src/e2e-test/resources/pluginParameters.properties

```diff
@@ -0,0 +1,84 @@
+#SFMC Credentials
+admin.clientid = SALESFORCE_MARKETING_CLIENT_ID
+admin.clientsecret =SALESFORCE_MARKETING_CLIENT_SECRET
+admin.base.uri=SALESFORCE_MARKETING_BASE_URI
+admin.soap.endpoint=SALESFORCE_MARKETING_SOAP_API_ENDPOINT
+admin.rest.endpoint = SALESFORCE_MARKETING_REST_API_ENDPOINT
+
+#SFMC User -Invalid Credentials
+invalid.clientid=a231
+invalid.clientsecret=564avf
+invalid.authenticationbase.uri=http//authenticationbase.com/
+invalid.soapapi.endpoint=http//invalidsoapi/
+invalid.restapibase.uri=http//invalidrespapi
+
+filter.value=folderId = 27860
+StoresId=xLko431
+
+#macro properties
+singleobjectmode.objectname=Mailing List
+multiobjectmode.objectlist=Mailing List
+singleobjectmode.dataextensionkey=Stores
+multiobjectmode.dataextensionkeylist=key1,key2
+multiobjectmode.tablename=Table1
+
+
+#Sink properties
+INSERT=insert
+UPDATE=update
+UPSERT=upsert
+TRUE=true
+FALSE=false
+sink.dataextensionkey=key1234
+sink.maxbatchsize=1000
+sink.columnmapping=inputfiled,dataextension
+
+#bq properties
+projectId=cdf-athena
```

Comment:
can we get projectId, datasetprojectId from the environment, similar to SFMC endpoints and credentials? This will make it easier to migrate these tests to run against a different project if needed. Also, it's preferable not to expose project names on public github repo.

Reply:
Sure we can do it. I have updated the Github Actions and Added Project Id and Dataset in GCP secrets. Now we are getting them from Environment in all the tests.

Also had to create a new method to drop the table. (https://github.com/data-integrations/salesforce-marketing/pull/34/commits/9bff48a06d63e875c4efa2c7ce37fe85eca8f3a3#diff-6cafbcddecea707397feb9d1901df1948a5bd07a82e11d564307d139c3c95e33R151)